bigtable/spark/src/main/scala/example/CopyTable.scala

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Please find the format for headers here (same comment for other files): https://github.com/GoogleCloudPlatform/java-docs-samples#source-code-headers

Reply:
Made some changes. Can you check out the latest version of the PR? Thanks.